process-automation/latest/model/interfaces.md

```diff
@@ -2,40 +2,60 @@
+The user interfaces (UI) section sets an end user interface for users to interact with content, tasks, and processes for the project using [Alfresco Digital Workspace]({% link process-automation/latest/using/index.md %}).
 
 ## Properties
 
-The basic properties of a user interface are:
+The basic properties of a UI are:
 
 | Property | Description |
 | -------- | ----------- |
 | UI name | *Required.* The name of the interface. Must be in lowercase and between 1 and 26 characters in length. Alphanumeric characters and hyphens are allowed, however the name must begin with a letter and end alphanumerically, for example `order-template`. |
-| Content/Process | *Required.* Select **Content** to force all the widgets in the Digital Workspace that refer to content, processes, and tasks to be displayed. Select **Process** when you want only the widgets that refer to processes, and tasks to be displayed. Additionally if **Process** is selected then the landing page can be configured that allows the user who runs the application to be redirected to the list of running processes, or to their list of My Tasks. |
+| Content and Process | *Required.* Select **Content** to force all the widgets in the Digital Workspace that refer to content, processes, and tasks to be displayed. Select **Process** when you want only the widgets that refer to processes, and tasks to be displayed. Additionally if **Process** is selected then the landing page can be configured that allows the user who runs the application to be redirected to the list of running processes, or to their list of My Tasks. |
 | UI description | *Optional.* A description of what the interface should be used for, for example `A template for orders to follow.` |
 | Default System Logo | *Optional.* The default system logo image. The recommended size for the logo is 28x28 px. **Note:** To set a new logo the file must be uploaded in advance with the visibility flag set to public, for more see [Files]({% link process-automation/latest/model/files.md %}). |
 | Application title | *Optional.* The name of the application. This is the name that will appear in the header and on the tab of your browser. |
 | Header text color HEX code | *Optional.* The color of the header text on the user interface using hexadecimal numbers. For example `#b39eba`. |
 | Header color HEX code | *Optional.* The color of the header background on the user interface using hexadecimal numbers. For example `#b39oba`. |
 | Default Background Header Image | *Optional.* The default background header image.  **Note:** To set a new background image the file must be uploaded in advance with the visibility flag set to public, for more see [Files]({% link process-automation/latest/model/files.md %}). |
 
-## Create a user interface
+## Create default UI
 
-To create a user interface:
+To create a default UI in JSON format:
```

Comment:
I don't think we should specify here that "default UI" is in JSON format. In fact, both - default and custom UI - are in JSON format.

Reply:
This has been done by @HYL-MURAWSKI  cheers.